`Web docs` - Moving content guidelines to tab (#3083)
hashicorp/design-system: website/docs/components/badge/partials/content/content.md

## General recommendations

> - Labels should be short and to the point (~25 characters). They should not consist of full sentences but should provide enough context to be useful, especially when using status badges.

Reviewer: ```suggestion
- Labels should be short and to the point (~25 characters). They should not consist of full sentences but should provide enough context to be useful, especially in the case of status badges.
```

Perhaps this is a bit clearer?
